contrib/cache_stdlibs.jl

```diff
@@ -23,13 +23,14 @@ stdlibs = [
     # 3-depth packages
     :LibGit2_jll,
 
+    # 4-depth packages
+    :SparseArrays,
+
     # 7-depth packages
     :LLD_jll,
-    :SuiteSparse_jll,
```

Comment:
That is still in `stdlib`?

Reply:
I thought it is a dependency of SparseArrays, so does not need to be specifically mentioned separately.

Reply:
Should they all be listed in here?

Reply:
Ideally yes, since this essentially does a "wave-front"  e.g. compiles all dependencies before it compiles the larger one.

I have a better infrastructure in https://github.com/JuliaLang/julia/pull/48069, but not enough time to work on that.